source/distributed/repartitioning_policy_tools.cc

```diff
@@ -206,10 +206,10 @@ namespace RepartitioningPolicyTools
 
     // step 1) check if all processes have enough cells
 
-    unsigned int n_locally_owned_active_cells = 0;
-    for (const auto &cell : tria_in.active_cell_iterators())
-      if (cell->is_locally_owned())
-        ++n_locally_owned_active_cells;
+    const unsigned int n_locally_owned_active_cells =
+      std::count_if(tria_in.begin_active(),
+                    tria_in.end(),
```

Comment:
```suggestion
                    typename Triangulation<dim, spacedim>::active_cell_iterator(
                      tria_in.end()),
```
This should fix the compiler problem
```
/home/runner/work/dealii/dealii/source/distributed/repartitioning_policy_tools.cc:210:20: note:   deduced conflicting types for parameter ‘_IIter’ (‘dealii::TriaActiveIterator<dealii::CellAccessor<1, 1> >’ and ‘dealii::TriaIterator<dealii::CellAccessor<1, 1> >’)
```